net/openssh/files/sshd.failsafe

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+failsafe_sshd () {
+
+	# if dropbear is executable it can handle failsafe
+	[[ -x /usr/sbin/dropbear ]] && return
+
+	sshd_tmpdir=/tmp/sshd
+	mkdir ${sshd_tmpdir}
+
+	sed -i 's/^root.*/root::0:17000:::::/g' /etc/shadow
+
+	for type in ed25519; do
+		key=${sshd_tmpdir}/ssh_host_${type}_key
+		ssh-keygen -N '' -t ${type} -f ${key}
+	done
+
+	mkdir -m 0700 -p /var/empty
+
+	cat > ${sshd_tmpdir}/sshd_config <<EOF
+
+HostKey ${sshd_tmpdir}/ssh_host_ed25519_key
+
+PermitRootLogin	yes
+PermitEmptyPasswords yes
```

Comment:
Wait... What?

Reply:
Also, if you're going to use multiple spaces, why not align RHS's?

Reply:
While in failsafe mode no password is set and openssh does not allow this which is why it is necessary. This is functionally equivalent to how dropbear works in regular failsafe. This whole patchset is only really useful if you want to run a dropbear-less system.